ironfish/jest.setup.env.js

```diff
@@ -35,4 +35,7 @@ beforeAll(() => {
 
 beforeEach(() => {
   consola.pause()
+  if (!process.env.GENERATE_TEST_REPORT) {
+    global.console = require('console')
```

Comment:
What's this used for?

Reply:
This is to not use jest console if we dont generate the report. In order to have the console log in the report, I have to config the jest to be verbose and the console log format is not efficient for human to read. It looks like the text below. 

When we are not generate the report, we can use the regular console and have it normal console output for local testing. 
```
src/mining/manager.test.perf.ts (21.361 s)
  ● Console

    console.log
      elapsed: 847.5135

      at Object.writeTestReport (src/testUtilities/utils.ts:31:13)

    console.log
      elapsed: 779.629041

      at Object.writeTestReport (src/testUtilities/utils.ts:31:13)

    console.log
      elapsed: 1894.2522920000001

      at Object.writeTestReport (src/testUtilities/utils.ts:31:13)

    console.log
      elapsed: 1564.0901669999998

      at Object.writeTestReport (src/testUtilities/utils.ts:31:13)

    console.log
      elapsed: 640.13825

      at Object.writeTestReport (src/testUtilities/utils.ts:31:13)

 PASS   @ironfish/sdk  src/blockchain/blockchain.test.perf.ts (81.282 s)
  ● Console

    console.info
      Added block seq: 20, hash: a9e57...07ef2, txs: 2, progress: 59.93%, time: 49.4ms

      at ConsoleReporter.logText (src/logger/reporters/console.ts:60:53)
          at async Promise.all (index 0)

    console.info
      Added block seq: 20, hash: 978fa...ef6bf, txs: 2, progress: 59.93%, time: 38.2ms

      at ConsoleReporter.logText (src/logger/reporters/console.ts:60:53)
          at async Promise.all (index 1)

    console.info
      Added block seq: 40, hash: 93a97...8039c, txs: 2, progress: 59.93%, time: 57.7ms

      at ConsoleReporter.logText (src/logger/reporters/console.ts:60:53)
          at async Promise.all (index 0)

    console.info
      Added block seq: 40, hash: 564bc...b6125, txs: 2, progress: 59.93%, time: 37.9ms

      at ConsoleReporter.logText (src/logger/reporters/console.ts:60:53)
          at async Promise.all (index 1)

    console.info
      Added block seq: 60, hash: a1cde...cf590, txs: 2, progress: 59.94%, time: 43.4ms

      at ConsoleReporter.logText (src/logger/reporters/console.ts:60:53)
          at async Promise.all (index 1)

    console.info
      Added block seq: 60, hash: 2ed81...a7d83, txs: 2, progress: 59.94%, time: 97.8ms

      at ConsoleReporter.logText (src/logger/reporters/console.ts:60:53)
          at async Promise.all (index 0)

    console.info
      Added block seq: 80, hash: 9f445...e0165, txs: 2, progress: 59.94%, time: 62.1ms

      at ConsoleReporter.logText (src/logger/reporters/console.ts:60:53)
          at async Promise.all (index 1)

    console.info
      Added block seq: 80, hash: 3ba30...907ad, txs: 2, progress: 59.94%, time: 42.3ms

      at ConsoleReporter.logText (src/logger/reporters/console.ts:60:53)
          at async Promise.all (index 0)

    console.info
      Added block seq: 100, hash: aa916...ff4fb, txs: 2, progress: 60.03%, time: 85.4ms

      at ConsoleReporter.logText (src/logger/reporters/console.ts:60:53)
          at async Promise.all (index 0)

    console.info
      Added block seq: 100, hash: be7fc...c0a9a, txs: 2, progress: 60.03%, time: 59.2ms

      at ConsoleReporter.logText (src/logger/reporters/console.ts:60:53)
          at async Promise.all (index 1)

    console.log
      TotalTestAverage: 23.20,InsertBlocksLinear: 0.00,InsertBlocksOnFork: 0.00,AddHeadRewindForkBlocks: 23.20

      at Object.writeTestReport (src/testUtilities/utils.ts:31:13)

    console.log
      TotalTestAverage: 160.00,InsertBlocksLinear: 26.70,InsertBlocksOnFork: 22.10,AddHeadRewindForkBlocks: 62.40

      at Object.writeTestReport (src/testUtilities/utils.ts:31:13)

    console.log
      TotalTestAverage: 259.00,InsertBlocksLinear: 28.40,InsertBlocksOnFork: 28.90,AddHeadRewindForkBlocks: 29.80

      at Object.writeTestReport (src/testUtilities/utils.ts:31:13)
```